src/iiif/parsing.py

```diff
@@ -85,9 +85,11 @@ def get_email_address(request, jwt_token):
         email_address = request.get_token_subject
     elif '@' in jwt_token.get('sub', ''):
         email_address = jwt_token['sub']
+    elif '@' in request.get_token_claims.get('email', ''):
```

Comment:
Is 't mogelijk dat er zowel in de jwt alsin keycloak een emailadres aanwezig is? Welke moet je dan hebben?

Reply:
Nee, in de authz token zit nooit een email. Plus, authz wordt uitgefaseerd, dus dit wordt sowieso dode code.

Reply:
En met authz bedoel je de die code mbt jwt? Als die code niet wordt gebruikt/wordt uitgefaseerd, kun je die dan ook niet beter weghalen? En anders een comment oid erbij?